app/commands/user/challenges/featured_exercises_progress_48_in_24.rb

```diff
@@ -73,7 +73,7 @@ def status(exercise)
     completed_exercises = completions[exercise[:slug]].to_a
     return :in_progress if completed_exercises.blank?
 
-    num_completions_in_2024 = completed_exercises.count { |(_, year)| year == 2024 }
+    num_completions_in_2024 = completed_exercises.count { |(_, date)| date[0..3] == '2024' || date == '2025-01-01' || date == '2023-12-31' }
```

Comment:
I expanded the year by a day in each direction to forgive timezones (as has been done for the monthly 12in23 badges)